source/wifi/wifi_hal.c

```diff
@@ -1032,8 +1032,6 @@ INT wifi_getRadioEnable(INT radioIndex, BOOL *output_bool)      //RDKB
                *output_bool=TRUE;
 
         //TODO: check if hostapd with config is running
-
-        *output_bool = FALSE;
```

Comment:
Can you please move this line to line 1028 ie before _sprintf(cmd,"ifconfig %s%d | grep RUNNING ", AP_PREFIX, radioIndex);_

Reply:
Moved a little bit earlier to cover the error case as well